stroeer/page/article/v1/article_page.proto

```diff
@@ -15,10 +15,8 @@ message ArticlePage {
   // Unpublished elements will be filtered according to their metadata.
   stroeer.core.v1.Article article = 1;
 
-  // Additional related editorial articles. Articles which are not published
-  // according to their `Metadata` will be filtered.
-  // deprecated in favor `core.Article.relatedArticles`
-  repeated RelatedArticle related_articles = 2 [deprecated = true];
+  // formerly related_articles
+  reserved 2;
```

Comment:
guess we don't need this reservation since all usages of this message are transient and we handle it internally?